lab_2_keywords_cooccurrence/main.py

```diff
@@ -3,12 +3,25 @@
 Extract keywords based on co-occurrence frequency
 """
 from pathlib import Path
-from typing import Optional, Sequence, Mapping
+from typing import Optional, Sequence, Mapping, Any
+from string import punctuation
+import json
 
 KeyPhrase = tuple[str, ...]
 KeyPhrases = Sequence[KeyPhrase]
 
 
+def correct_type(variable: Any, expected_type: type) -> bool:
+    """
+    Checks the type of variable
+    """
+    if not isinstance(variable, expected_type) or not variable:
+        return False
+    if isinstance(variable, int) and variable < 0:
```

Comment:
return the result of the `isinstance(variable, int) and variable >= 0`

Reply:
it doesn't work

Reply:
yes, because you need to add `not` -> `not (isinstance(variable, int) and variable < 0)` or `not isinstance(variable, int) or variable >= 0`